templates/package.json

```diff
@@ -43,7 +43,9 @@
         "tab/js/non-sso",
         "tab/ts/non-sso",
         "tab/js/m365",
-        "tab/ts/m365"
+        "tab/ts/m365",
+        "tab/ts/dashboard",        
+        "tab/js/dashboard"
```

Comment:
nothing serious, but better to swap line 47 & 48, as there is always js first and ts second in this list

Reply:
Thanks, I've adjusted their order.